src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,56 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 크기를 확인 할 수 있다")
+    @Test
+    void checkSize() {
+        //given
+        int expected = 3;
+
+        //when
+        int actual = numbers.size();
+
+        //then
+        assertThat(actual).isEqualTo(expected);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
given/when/then으로 나누지 말고 위와 같이 한 라인으로 구현하는 것은 어떨까?
given/when/then을 분리하다보니 오히려 라인 수가 늘어나고 구현할 코드, 가독성도 떨어지는 것은 아닐까?